docker-compose.yml

```diff
@@ -56,6 +56,19 @@ services:
       - "2379:2379"
     networks:
       - default
+  pattern-pedia-auth:
+    build: https://github.com/PatternAtlas/pattern-pedia-auth.git
+    environment:
+      JDBC_DATABASE_URL: db
+      JDBC_DATABASE_USERNAME: postgres
```

Comment:
can you adopt this postgres user as well? we chose to use patternatlas instead of postgres everywhere :)

Reply:
Done